lib/main.dart

```diff
@@ -1,23 +1,35 @@
 import 'package:flutter/material.dart';
-// import 'package:langx_flutter/pages/home/bottom_navigation_bar.dart';
+import 'package:appwrite/appwrite.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
 import 'package:langx_flutter/login.dart';
-// import 'package:langx_flutter/old/community.dart';
-// import 'package:langx_flutter/pages/home/profile.dart';
-// import 'package:langx_flutter/pages/home/profile.dart';
-// import 'package:langx_flutter/pages/home/home.dart';
 
-void main() {
-  runApp(const Main());
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await dotenv.load();
+
+  Client client = Client()
+      .setEndpoint(dotenv.env['APP_ENDPOINT']!)
+      .setProject(dotenv.env['APP_PROJECT']!);
+  Account account = Account(client);
+
+  runApp(Main(
+    home: LoginScreen(account: account),
+    account: account,
+  ));
 }
 
 class Main extends StatelessWidget {
-  const Main({super.key});
+  final Widget home;
```

Comment:
What's this for?

Reply:
Its also fixing warning that flutter analyze has, for more 'use_super_parameters' you can take a look